### PR TITLE
Marketplace Billing integration

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by lbadenhop** [comment]
@cbordier Ready for your review. Content is from https://gist.github.com/lbrenman/b7364325e208c11fcb8f738a2bcab4b0, but I've organized it a little differently than in the original document. Please let me know if content should be added / removed or reorganized. Thanks

File: content/en/docs/manage_marketplace/billing_integration/install_configure_billing_integration.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    ![Product Plan YAML](/Images/marketplace/billing_integration/config13.png)
+
+2. Modify the [Amplify Subscription Integration YAML file](https://gist.github.com/lbrenman/ba8733b78141e0ebf6de88404af12625#file-monitor-updated-deleted-subscriptions-integration-yaml) with the URL of your HTTP/S Post Webhook for your *provision subscription*.
+
+    ![Subscription YAML](/Images/marketplace/billing_integration/config14.png)
+
+3. Create your [Amplify integration](https://blog.axway.com/product-insights/amplify-platform/central/create-an-amplify-central-integration-webhook-using-the-axway-cli) using the CLI:
+
+    ```yaml
+    amplify central create -f monitor-updated-deleted-plans-integration.yaml
```
Comment (cbordier) [inline content/en/docs/manage_marketplace/billing_integration/install_configure_billing_integration.md:77]: replace amplify by axway

File: content/en/docs/manage_marketplace/billing_integration/install_configure_billing_integration.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    ```
+
+    and
+
+    ```yaml
+    amplify central create -f monitor-updated-deleted-subscriptions-integration.yaml
```
Comment (cbordier) [inline content/en/docs/manage_marketplace/billing_integration/install_configure_billing_integration.md:83]: replace amplify by axway

File: content/en/docs/manage_marketplace/billing_integration/install_configure_billing_integration.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+![billingSystem recurly](/Images/marketplace/billing_integration/config22.png)
+
+![Product Foundry](/Images/marketplace/billing_integration/config23.png)
+
+The product plan must be a paid plan with manual approval. Only product plans with at least one Tiered/Graduated and/or Pay Per Use quota are supported.
+
+![Tiered Graduated](/Images/marketplace/billing_integration/config24.png)
+
+For product subscriptions to be linked to subscriptions in the billing platform, such as Recurly, an account must exist in in the billing platform with a code that corresponds to the subscriber's user ID, team ID or organization ID.
+
```
Comment (cbordier) [inline content/en/docs/manage_marketplace/billing_integration/install_configure_billing_integration.md:147]: Add a samples to help user understand the above sentence meaning:
Amplify team ID | Recurly account code |
-------------------|-------------------------|
ABCD                  | amplify-team-ABCD    |

Amplify user ID | Recurly account code |
-------------------|-------------------------|
ABCD                  | amplify-user-ABCD    |

File: content/en/docs/manage_marketplace/billing_integration/webhook_authentication.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+---
+title: Integration webhook authentication
+linkTitle: webhook authentication
```
Comment (cbordier) [inline content/en/docs/manage_marketplace/billing_integration/webhook_authentication.md:3]: I think the link title should be Integration webhook authentication to match the TOC.

File: content/en/docs/manage_marketplace/billing_integration/billing_integration_use.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+    ![Recurly accounts details](/Images/marketplace/billing_integration/usage8.png)
+
+6. Once the plan and subscription are linked between Amplify and the billing platform, the integration sends API traffic data periodically (nightly / hourly) to the subscription for proper billing.
+
+    ![billing details](/Images/marketplace/billing_integration/usage9.png)
+
+**Usage Notes**:
+
+* To specify which product plans should be linked to the billing platform, the API provider must add a specific attribute to those Products with either the API, CLI or the UI.
```
Comment (cbordier) [inline content/en/docs/manage_marketplace/billing_integration/billing_integration_use.md:53]: To specify which product plans should be linked to the billing platform, the API provider must add a specific attribute (billingSystem=recurly) to those Products with either the API, CLI or the UI.

**Comment by lbadenhop** [comment]
@cbordier Your review comments have been incorporated.

**Pull request merged**